pkg_pipeline/single_build.R

```diff
@@ -0,0 +1,167 @@
+#! /usr/local/bin/Rscript
+
+success <- c(
+    check = FALSE,
+    test = FALSE,
+    lint = FALSE,
+    valdoc = FALSE
+)
+
+# Getting package information ----
+pkg_name <- read.dcf("DESCRIPTION")[1, "Package"]
+pkg_version <- read.dcf("DESCRIPTION")[1, "Version"]
+
+# Building ----
+
+message("############################")
+message("###### INSTALLING (S) ######")
+message("############################")
+
+pak::local_install_deps(".", upgrade=FALSE, ask=FALSE, dependencies = TRUE)
```

Comment:
I would say keep the deps step removed above (since it's a dependency in all steps, not just check/lintr/pkgdown) and comment these 2 lines

Reply:
also why not add the qc report code in this repo in the first place? it uses nobody outside BI to have it in the package repo

Reply:
The qc report is used inside the test files to link specs and test cases, therefore it is required when running tests. In a next iteration we can think about ways to move it to dv.templates. No problem with that.

Reply:
why not do it in the first place?